rcl/src/rcl/logging.c

```diff
@@ -100,7 +101,8 @@ rcl_logging_configure_with_output_handler(
     }
   }
   if (g_rcl_logging_ext_lib_enabled) {
-    status = rcl_logging_external_initialize(config_file, g_logging_allocator);
+    status = rcl_logging_external_initialize(
+      file_name_prefix, config_file, g_logging_allocator);
```

Comment:
since `file_name_prefix` is in the global arguments, i do not think upstream packages need to care about this change as API/ABI. but probably there are some tests using `--log-level` in `launch`, `ros2cli` and so on, those should be updated with new test cases with `--log-file-name`. (besides `rclc` needs to be checked.)